### ISSUE TITLE
Segmentation fault

### ISSUE THREAD

**Issue by SenaChanghoon** [TexasInstruments/edgeai-modelmaker#9]
I have problems training and compiling by using run_modelmaker.sh.

After training the model with custom dataset or the default dataset which is in the config_detection.yaml,
the following error occurs at the end.

INFO:20230117-104719: import  - modelartifacts - this may take some time..../run_modelmaker.sh: line 38: 486218 Segmentation fault      (core dumped) python ./scripts/run_modelmaker.py $@

Any suggestions?

I made a docker environment(Ubuntu 18.04, CUDA 11.3) on my GPU sever

pip list
absl-py                        1.3.0
addict                         2.4.0
appdirs                        1.4.4
asgiref                        3.4.1
attr                           0.3.1
attrs                          22.2.0
azure-core                     1.24.2
azure-storage-blob             12.13.1
boto                           2.49.0
boto3                          1.16.28
botocore                       1.19.28
boxing                         0.1.4
cached-property                1.5.2
cachetools                     4.2.4
certifi                        2022.12.7
cffi                           1.15.1
charset-normalizer             2.0.12
click                          8.0.4
cloudpickle                    2.2.0
colorama                       0.4.5
coreapi                        2.3.3
coreschema                     0.0.4
cryptography                   39.0.0
cycler                         0.11.0
Cython                         0.29.33
dataclasses                    0.8
decorator                      5.1.1
distro                         1.8.0
Django                         3.1.13
django-annoying                0.10.6
django-cors-headers            3.6.0
django-debug-toolbar           3.2.1
django-extensions              3.1.0
django-filter                  2.4.0
django-model-utils             4.1.1
django-ranged-fileresponse     0.1.2
django-redis-cache             3.0.0
django-rest-swagger            2.2.0
django-rq                      2.4.1
django-user-agents             0.4.0
djangorestframework            3.13.1
dlr                            1.10.0
drf-dynamic-fields             0.3.0
drf-flex-fields                0.9.5
drf-generators                 0.3.0
drf-yasg                       1.20.0
edgeai-benchmark               8.4.0.12+78a94b8         /opt/beagle/edgeai-benchmark
flatbuffers                    1.12
google-api-core                2.8.2
google-auth                    1.35.0
google-auth-oauthlib           0.4.6
google-cloud-appengine-logging 0.2.0
google-cloud-audit-log         0.2.2
google-cloud-core              1.7.3
google-cloud-logging           2.6.0
google-cloud-storage           1.28.1
google-resumable-media         0.5.1
googleapis-common-protos       1.56.3
graphviz                       0.19.1
grpcio                         1.48.2
grpcio-status                  1.48.2
h5py                           3.1.0
htmlmin                        0.1.12
idna                           3.4
importlib-metadata             4.8.3
importlib-resources            5.4.0
inflection                     0.5.1
iniconfig                      1.1.1
isodate                        0.6.1
itypes                         1.2.0
Jinja2                         3.0.3
jmespath                       0.10.0
joblib                         1.1.1
json-tricks                    3.16.1
jsonschema                     3.2.0
kiwisolver                     1.3.1
label-studio                   1.4
label-studio-converter         0.0.36
lockfile                       0.12.2
lxml                           4.9.2
Markdown                       3.3.7
MarkupSafe                     2.0.1
matplotlib                     3.3.4
mmcv-full                      1.4.7
mmdet                          2.22.0                   /opt/beagle/edgeai-mmdetection
msrest                         0.7.1
munkres                        1.1.4
natsort                        8.2.0
ninja                          1.11.1
nltk                           3.5
numpy                          1.19.5
oauthlib                       3.2.2
onnx                           1.8.1
onnxruntime-tidl               1.7.0
openapi-codec                  1.3.2
opencv-python                  4.2.0.34
ordered-set                    4.0.2
osrt-model-tools               1.2
packaging                      21.3
pandas                         1.1.5
Pillow                         8.3.2
Pillow-SIMD                    7.2.0.post1
pip                            21.3.1
pluggy                         1.0.0
plyfile                        0.7.4
progiter                       1.0.1
proto-plus                     1.22.2
protobuf                       3.19.0
psutil                         5.9.4
psycopg2-binary                2.9.1
py                             1.11.0
pyasn1                         0.4.8
pyasn1-modules                 0.2.8
pycocotools                    2.0.6
pycparser                      2.21
pydantic                       1.8.2
pyparsing                      3.0.9
pyrsistent                     0.18.0
pytest                         7.0.1
python-dateutil                2.8.1
pytz                           2019.3
PyYAML                         6.0
redis                          3.5.3
regex                          2022.10.31
requests                       2.27.1
requests-oauthlib              1.3.1
rq                             1.10.0
rsa                            4.9
ruamel.yaml                    0.17.21
ruamel.yaml.clib               0.2.7
rules                          2.2
s3transfer                     0.3.7
scikit-learn                   0.24.2
scipy                          1.5.4
seaborn                        0.11.2
sentry-sdk                     1.13.0
setuptools                     59.6.0
simplejson                     3.18.1
six                            1.16.0
sqlparse                       0.4.3
synr                           0.6.0
tensorboard                    2.10.1
tensorboard-data-server        0.6.1
tensorboard-plugin-wit         1.8.1
terminaltables                 3.1.10
tflite                         2.10.0
tflite-runtime                 2.8.2
thop                           0.1.1.post2209072238
threadpoolctl                  3.1.0
tomli                          1.2.3
torch                          1.10.0+cu113
torchinfo                      1.5.4
torchvision                    0.11.0a0+4b649a8         /opt/beagle/edgeai-torchvision
tornado                        6.1
tqdm                           4.64.1
tvm                            1.11.1.dev335+g13a4007ca
typing_extensions              4.1.1
ua-parser                      0.16.1
ujson                          4.3.0
uritemplate                    4.1.1
urllib3                        1.26.13
user-agents                    2.2.0
Werkzeug                       2.0.3
wheel                          0.37.1
wurlitzer                      3.0.3
xmljson                        0.2.0
yapf                           0.32.0
zipp                           3.6.0

**Comment by SenaChanghoon** [comment]
silly me, I haven't installed wget so when I ran ./setup_all.sh I didn't saw https://software-dl.ti.com/jacinto7/esd/tidl-tools/08_04_00_00/tidl_tools.tar.gz wasn't downloaded.
